frontendUiRootImpl/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
   implementation project(':binding')
```

Comment:
Btw (unrel) if you could bump 2020.3 EAP version in build.gradle to the recent Beta as per https://www.jetbrains.com/intellij-repository/snapshots/ ... ofc if it fails for any reason then let's investigate separately